content/_index.de.md

```diff
@@ -10,12 +10,12 @@ title = "Das Postmeritokratische Manifest"
 
 Meritokratie ist ein grundlegendes Prinzip der Open-Source-Bewegung und das meritokratische Ideal wird im Bereich der Informationstechnologie (IT) durch die Art und Weise, in der Menschen angeworben, eingestellt, in Unternehmen gehalten, befördert und wertgeschätzt werden, fortwährend aufrechterhalten.
```

Comment:
I like the new sentence structure and agree that it sounds more fluid in German :+1:
I also really like the connection between the German translation and the original wording `merit` here. 
I'm wondering if it might make sense to try to underline that `merit` translates to `das Verdienst` in contrast to `der Verdienst` here, too. If `das Verdienst` is used in its plural form, the distinction between `das Verdienst` and `der Verdienst` gets lost. In my opinion it would be useful to make this distinction clear as soon as the term `Verdienst` is used for the first time in the text so the reader wouldn't be confused to find the term used together with the explicit article `das` later on in the text and then (eventually) wonder where the term comes from or if this was simply a typo and should rather be translated with `der Verdienst`. 

What do you think about this?

I imagine we could go with something like

```
[...] im Bereich der Informationstechnologie fortwährend aufrechterhalten: das Verdienst (engl. "the merit") eines Menschen wirkt sich auf dessen Anwerbung, Einstellung, Beförderung und Wertschätzung aus.
```

I think it'd be also valid to replace `ihren Verdiensten` simply with `anerkennenswerten Leistungen` to avoid using the less common term `das Verdienst`. I also tried to circumvent this issue [in the same way here](https://github.com/CoralineAda/postmeritocracy/pull/19#discussion_r191921970).

Reply:
Sounds really good to me! I can also stand behind replacing `das Verdienst` with `anerkennenswerte Leistungen`, as it makes the text more accessible.

Reply:
Yes, that's also a good point :+1: